webroot/js/app.js

```diff
@@ -383,9 +383,45 @@ export default class App extends Component {
     }
   }
 
+  handleMuteKeyPressed() {
+    this.player.vjsPlayer.muted(!this.player.vjsPlayer.muted());
```

Comment:
This doesn't seem to work if you start the player already muted.  Interesting, in these cases `this.player.vjsPlayer.muted()` returns false, even if it's muted.